hub/src/PeerController.cpp

```diff
@@ -39,6 +39,72 @@ void CPeerController::refresh() {
     });
 }
 
+const QString& CPeerController::status_description(const QString &status){
+    static std::map<QString, QString> dct_desp = {
+
+        {"aborted", tr ("The Peer is in an aborted state. This means that it was abruptly "
+                        "stopped without properly closing the session. Run `start` "
+                        "to resume this virtual machine. If any problems persist, you may "
+                        "have to destroy and restart the virtual machine.")},
```

Comment:
@Erkin97 I think it would sound better if we say this: "The Peer is aborted because it was abruptly stopped. Press `start` to resume the virtual machine. If you still would be facing the problem, you can destroy and restart it."